daphne/src/lib.rs

```diff
@@ -981,6 +927,7 @@ pub mod auth;
 pub mod constants;
 #[cfg(test)]
 mod constants_test;
+pub mod dap_error;
```

Comment:
I think this should be called "error", for consistency with the "abort" module.

Reply:
yeah I thought about that, but there are multiple errors defined in this crate (`TransitionFailure`, `DapAbort`, for example) and since each is defined in a different module it felt a bit misleading to have this be called `error`, as usually such a module houses all the errors of a crate.

Reply:
`TransitionFailure` is a protocol message and so is not an error as such. (It's never used as the error type of `Result`, IIRC.) 

What about merging `abort` and `dap_error` into a common module, called `error`?